servers/update_api_gateway.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Pull down the most recent versions of all containers
+docker-compose pull
+
+# Restart oudated running containers
```

Comment:
`docker-compose` will automatically check to see which images don't have a `latest` tag that matches what have been pulled. Then it will restart them